java-manta-client-unshaded/src/test/resources/testng.xml

```diff
@@ -62,13 +62,44 @@
         </classes>
     </test>
     <test name="Encryption Tests">
-        <packages>
-            <package name="com.joyent.manta.client.crypto.*" />
-        </packages>
         <classes>
             <class name="com.joyent.manta.http.EncryptedHttpHelperTest" />
             <class name="com.joyent.manta.http.MantaHttpHeadersByteRangeTest" />
             <class name="com.joyent.manta.http.MantaHttpHeadersTest" />
+            <class name="com.joyent.manta.client.crypto.ExternalSecurityProviderLoaderTest" />
```

Comment:
Since exclusions in the previous code were only possible at a package level, I added all the classes within the ```com.joyent.manta.client.crypto``` package excluding the new unit-test since it has a different testing configuration.